src/mame/machine/k573dio.cpp

```diff
@@ -266,6 +265,16 @@ READ16_MEMBER(k573dio_device::mpeg_ctrl_r)
 WRITE16_MEMBER(k573dio_device::mpeg_ctrl_w)
 {
 	k573fpga->set_mpeg_ctrl(data);
+
+	if (data == 0xe000) {
```

Comment:
The game doesn't seem to signal in any other way when playback should be stopped, so these flags need to be respected for playback to work properly

See also 7a55821cc800e59d48b11aa3dc8d521ca9c5e82d for some further fixes involving audio playback.

Reply:
Hmmm, I don't remember, does the mas have a way to be told to start or not?

Reply:
Good question. I looked over the datasheet again and it doesn't seem like there's an explicit command to start. I know it definitely doesn't have a stop command.

I guess I will try to rewrite this bit in a way that makes the FPGA stop sending data to the mas3507d instead of having the mas3507d handle it as a stop command then since that functionality would probably be more in line with what it's really doing (or should be doing).

Another theory I have is that it might be sending commands to set the volume to mute it instead of stopping sending data. The reason I think that might also be a possibility is because when I have my logic analyzer hooked up to the SID/SIC pins on the mas3507d, I can still see it receiving MP3 data even if you "stop" the current MP3 (for example, in the sound test menu, if you go from BGM test to another option and it stops playing the test MP3). I really don't want to get out the hardware again to verify that it's sending those kinds of commands over I2S though, so it will probably just remain a theory.

Reply:
See this comment: https://github.com/mamedev/mame/pull/5133#discussion_r290160859

The code has been rewritten to not have the concept of "start" and "stop" but still works.